resources/assets/lib/osu-audio/settings.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import { route } from 'laroute';
+import Main from './main';
+
+export default class Settings {
```

Comment:
nothing in this class seems to require being declared as an arrow function on a class field, regular function prototypes should be preferred; simple `get`/`set` can just use getters and setters